src/resources/views/crud/columns/table_relationships.blade.php

```diff
@@ -0,0 +1,41 @@
+@php
```

Comment:
I think a better name for the column itself would be `relationship_table` (inverted and singular). That because:
- it really only takes care of one relationship, not multiple (hence, singular);
- it's the proper order in the English language (`relationship table` not `table relationship`);
- it would be more consistent with another Backpack column - `relationship`; so you'd have `relationship` which outputs them inline and `relationship_table` which outputs them in a table; and I think that's cool;